lib/HLSL/HLOperations.cpp

```diff
@@ -421,21 +421,21 @@ HLBinaryOpcode GetUnsignedOpcode(HLBinaryOpcode opcode) {
 
 static void SetHLFunctionAttribute(Function *F, HLOpcodeGroup group,
                                        unsigned opcode) {
+  F->addFnAttr(Attribute::NoUnwind);
```

Comment:
I'd like to understand better why this can be unconditionally included where it wasn't before

Reply:
I think we just added them as needed. Some of the HLOpcodeGroup's get replaced up pretty early with Matrix lowering, so inability to do DCE wouldn't really have impacted them very much.